x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/lucene/TimeSeriesSortedSourceOperatorFactory.java

```diff
@@ -261,29 +261,20 @@ void consume() throws IOException {
                     }
                 } else {
                     int previousTsidOrd = leaf.timeSeriesHashOrd;
-                    boolean breakOnNextTsidChange = false;
```

Comment:
I think this will be fixed when the other pr you opened is merged?

Reply:
Yes, I have merged this fix in another PR.